webapp-django/crashstats/settings/base.py

```diff
@@ -723,7 +723,7 @@ def path(*dirs):
         },
         'postgresql': {
             'database_hostname': config(
-                'DATABASE_HOSTNAME',
+                'DATABASE_HOST',
                 'localhost'
             ),
```

Comment:
This seems messy to me because it's changed the environment variable, but the configuration variable is still `database_hostname` so the two don't match. Matching is nice since it's less cognitive load.

Is there a reason we can't change the configuration variable, too? Or am I misunderstanding what's going on here?